library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -12,6 +13,15 @@
 
     .rubric-item {
         position: relative;
+
+        .divDescription{
+            display: inherit;
+        }
+    
+        .pDescription{
+            width: min-content;
+            float: left;
```

Comment:
Thanks for the review @profmikegreene , I've marked this PR as work in progress some hours ago because we expect more work here, I'm not convinced with the implementation.